### PR TITLE
"Blackify" parts of the webapp

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

I don't believe these things should be "red". Also, re-implement the
transition on the back link on pages hehe :see-no-evil:

---

Also fix a unrelated minor bug; [Show skeleton user cells when fetching registrations](https://github.com/webkom/lego-webapp/commit/25384b1e6469219e94453795b1171023332dd23a)

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

<table>
    <tr>
		<td width="25%">Description</td>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
		<td>Back link, notification badge and hamburger icon</td>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

Looked at the changes - and they looked alright 👍🏼

### CODE REVIEW

File: app/routes/bdb/components/OptionsBox.tsx [hunk 1]
```diff
@@ -99,8 +99,6 @@ const OptionsBox = ({ updateFilters, removeFilters }: Props) => {
               name="studentContact"
               filter={[AutocompleteContentType.User]}
               onChange={(user: AutocompleteUser) => {
-                console.log(studentContactValue);
-                console.log(user);
```
Comment (ivarnakken) [inline app/routes/bdb/components/OptionsBox.tsx:103]: @eikhr did you need these?

**Review by falbru — APPROVED** [review]
Looks good!

**Review by norbye — APPROVED** [review]

**Review by itsisak — COMMENTED** [review]
Why should the back button not have `lego-link-color`? Is it reserved for external links or something?

**Comment by ivarnakken** [comment]
> Is it reserved for external links or something?

No not at all

> Why should the back button not have lego-link-color?

I don't know! haha
I couldn't sleep tonight thinking about it. I think I'll change it back to `lego-link-color` 😆

**Review by itsisak — APPROVED** [review]

**Pull request merged**